tilecloud_chain/controller.py

```diff
@@ -224,7 +224,7 @@ def _generate_legend_images(gene):
                     for l in layer['layers'].split(','):
                         response = session.get(layer['url'] + '?' + urlencode({
                             'SERVICE': 'WMS',
-                            'VERSION': '1.1.1',
+                            'VERSION': layer['version'],
```

Comment:
should also be modified here: https://github.com/camptocamp/tilecloud-chain/blob/master/tilecloud_chain/server.py#L311